### DISCUSSION TITLE
Error installation

### DISCUSSION THREAD

**Discussion by Disc0-0** [gllmlbrt/Alsavo-HassOS#2] [General]
Thanks for your work, cool to get heatpump into Hassio!

I got an error when running:
apk add cmake make musl-dev build-base linux-headers curl-dev

ERROR: unable to select packages:
  musl-1.2.4-r0:
    breaks: musl-dev-1.2.4-r1[musl=1.2.4-r1]
    satisfies: world[musl=1.2.4-r0] apk-tools-2.14.0-r2[musl>=1.2.3_git20230424]
               apk-tools-2.14.0-r2[so:libc.musl-x86_64.so.1] libbsd-0.11.7-r1[musl]
               libbsd-0.11.7-r1[so:libc.musl-x86_64.so.1] isl26-0.26-r1[so:libc.musl-x86_64.so.1]
               pkgconf-1.9.5-r0[so:libc.musl-x86_64.so.1] brotli-1.0.9-r14[so:libc.musl-x86_64.so.1]
               libblkid-2.38.1-r8[so:libc.musl-x86_64.so.1] rsync-3.2.7-r4[so:libc.musl-x86_64.so.1]
               dash-0.5.12-r2[so:libc.musl-x86_64.so.1]
               gcc-12.2.1_git20220924-r10[so:libc.musl-x86_64.so.1]
               wget-1.21.4-r0[so:libc.musl-x86_64.so.1] libpcap-1.10.4-r1[so:libc.musl-x86_64.so.1]
               perl-5.36.1-r2[so:libc.musl-x86_64.so.1] alsa-lib-1.2.9-r1[so:libc.musl-x86_64.so.1]
               nss-3.91-r0[so:libc.musl-x86_64.so.1]
               g++-12.2.1_git20220924-r10[so:libc.musl-x86_64.so.1]
               libffi-3.4.4-r2[so:libc.musl-x86_64.so.1] alsa-utils-1.2.9-r0[so:libc.musl-x86_64.so.1]
               libevent-2.1.12-r6[so:libc.musl-x86_64.so.1] bluez-5.66-r5[so:libc.musl-x86_64.so.1]
               bind-tools-9.18.16-r0[so:libc.musl-x86_64.so.1] nmap-7.93-r1[so:libc.musl-x86_64.so.1]
               patch-2.7.6-r10[so:libc.musl-x86_64.so.1] json-c-0.16-r3[so:libc.musl-x86_64.so.1]
               mii-tool-2.10-r3[so:libc.musl-x86_64.so.1] bash-5.2.15-r5[so:libc.musl-x86_64.so.1]
               gcompat-1.1.0-r1[so:libc.musl-x86_64.so.1] xxd-9.0.1568-r0[so:libc.musl-x86_64.so.1]
               unzip-6.0-r14[so:libc.musl-x86_64.so.1]
               ca-certificates-20230506-r0[so:libc.musl-x86_64.so.1]
               fstrm-0.6.1-r2[so:libc.musl-x86_64.so.1] python3-3.11.4-r0[so:libc.musl-x86_64.so.1]
               mariadb-client-10.11.4-r0[so:libc.musl-x86_64.so.1]
               libxau-1.0.11-r2[so:libc.musl-x86_64.so.1] binutils-2.40-r7[so:libc.musl-x86_64.so.1]
               net-tools-2.10-r3[so:libc.musl-x86_64.so.1]
               lua5.3-libs-5.3.6-r5[so:libc.musl-x86_64.so.1]
               busybox-1.36.1-r0[so:libc.musl-x86_64.so.1] less-633-r0[so:libc.musl-x86_64.so.1]
               openssh-client-common-9.3_p2-r0[so:libc.musl-x86_64.so.1]
               lz4-libs-1.9.4-r4[so:libc.musl-x86_64.so.1]
               bind-libs-9.18.16-r0[so:libc.musl-x86_64.so.1]
               sqlite-libs-3.41.2-r2[so:libc.musl-x86_64.so.1] libmd-1.0.4-r2[so:libc.musl-x86_64.so.1]
               rhash-libs-1.4.3-r2[so:libc.musl-x86_64.so.1]
               libucontext-1.2-r2[so:libc.musl-x86_64.so.1] libpsl-0.21.2-r0[so:libc.musl-x86_64.so.1]
               keyutils-libs-1.6.3-r3[so:libc.musl-x86_64.so.1]
               sudo-1.9.13_p3-r2[so:libc.musl-x86_64.so.1] curl-8.2.0-r1[so:libc.musl-x86_64.so.1]
               nspr-4.35-r2[so:libc.musl-x86_64.so.1] eudev-libs-3.2.11-r8[so:libc.musl-x86_64.so.1]
               sqlite-3.41.2-r2[so:libc.musl-x86_64.so.1] libcap2-2.69-r0[so:libc.musl-x86_64.so.1]
               libvorbis-1.3.7-r1[so:libc.musl-x86_64.so.1] vim-9.0.1568-r0[so:libc.musl-x86_64.so.1]
               krb5-libs-1.20.1-r1[so:libc.musl-x86_64.so.1]
               flac-libs-1.4.3-r0[so:libc.musl-x86_64.so.1] libuv-1.44.2-r2[so:libc.musl-x86_64.so.1]
               libx11-1.8.4-r4[so:libc.musl-x86_64.so.1] libxcb-1.15-r1[so:libc.musl-x86_64.so.1]
               mosh-client-1.4.0-r1[so:libc.musl-x86_64.so.1]
               libpanelw-6.4_p20230506-r0[so:libc.musl-x86_64.so.1]
               pwgen-2.08-r3[so:libc.musl-x86_64.so.1] git-2.40.1-r0[so:libc.musl-x86_64.so.1]
               soxr-0.1.3-r5[so:libc.musl-x86_64.so.1]
               fftw-single-libs-3.3.10-r2[so:libc.musl-x86_64.so.1]
               pulseaudio-utils-16.1-r10[so:libc.musl-x86_64.so.1]
               libsndfile-1.2.0-r2[so:libc.musl-x86_64.so.1] tdb-libs-1.4.8-r1[so:libc.musl-x86_64.so.1]
               zsh-5.9-r2[so:libc.musl-x86_64.so.1] musl-utils-1.2.4-r0[so:libc.musl-x86_64.so.1]
               nghttp2-libs-1.55.1-r0[so:libc.musl-x86_64.so.1]
               diffutils-3.9-r2[so:libc.musl-x86_64.so.1] cjson-1.7.15-r4[so:libc.musl-x86_64.so.1]
               libstdc++-12.2.1_git20220924-r10[so:libc.musl-x86_64.so.1]
               mpdecimal-2.5.1-r2[so:libc.musl-x86_64.so.1] dbus-1.14.8-r0[so:libc.musl-x86_64.so.1]
               libbz2-1.0.8-r5[so:libc.musl-x86_64.so.1] brotli-libs-1.0.9-r14[so:libc.musl-x86_64.so.1]
               libcom_err-1.47.0-r2[so:libc.musl-x86_64.so.1] libacl-2.3.1-r3[so:libc.musl-x86_64.so.1]
               file-5.45-r0[so:libc.musl-x86_64.so.1] libogg-1.3.5-r4[so:libc.musl-x86_64.so.1]
               openssh-sftp-server-9.3_p2-r0[so:libc.musl-x86_64.so.1]
               libcurl-8.2.0-r1[so:libc.musl-x86_64.so.1]
               networkmanager-common-1.42.8-r0[so:libc.musl-x86_64.so.1]
               mpc1-1.3.1-r1[so:libc.musl-x86_64.so.1]
               libmenuw-6.4_p20230506-r0[so:libc.musl-x86_64.so.1]
               libnm-1.42.8-r0[so:libc.musl-x86_64.so.1] mosh-server-1.4.0-r1[so:libc.musl-x86_64.so.1]
               libndp-1.8-r1[so:libc.musl-x86_64.so.1] libmagic-5.45-r0[so:libc.musl-x86_64.so.1]
               cmake-3.26.5-r0[so:libc.musl-x86_64.so.1]
               libxml2-utils-2.11.4-r0[so:libc.musl-x86_64.so.1]
               musl-obstack-1.2.3-r2[so:libc.musl-x86_64.so.1] gdbm-1.23-r1[so:libc.musl-x86_64.so.1]
               ncurses-6.4_p20230506-r0[so:libc.musl-x86_64.so.1]
               libpulse-16.1-r10[so:libc.musl-x86_64.so.1] pcre-8.45-r3[so:libc.musl-x86_64.so.1]
               oniguruma-6.9.8-r1[so:libc.musl-x86_64.so.1] mpfr4-4.2.0_p12-r0[so:libc.musl-x86_64.so.1]
               tmux-3.3a_git20230428-r0[so:libc.musl-x86_64.so.1]
               libcrypto3-3.1.1-r3[so:libc.musl-x86_64.so.1] zlib-1.2.13-r1[so:libc.musl-x86_64.so.1]
               libmount-2.38.1-r8[so:libc.musl-x86_64.so.1] glib-2.76.4-r0[so:libc.musl-x86_64.so.1]
               zip-3.0-r12[so:libc.musl-x86_64.so.1]
               libgomp-12.2.1_git20220924-r10[so:libc.musl-x86_64.so.1]
               protobuf-c-1.4.1-r2[so:libc.musl-x86_64.so.1] nano-7.2-r1[so:libc.musl-x86_64.so.1]
               libprotobuf-3.21.12-r2[so:libc.musl-x86_64.so.1]
               libltdl-2.4.7-r2[so:libc.musl-x86_64.so.1]
               networkmanager-1.42.8-r0[so:libc.musl-x86_64.so.1]
               libintl-0.21.1-r7[so:libc.musl-x86_64.so.1] htop-3.2.2-r1[so:libc.musl-x86_64.so.1]
               readline-8.2.1-r1[so:libc.musl-x86_64.so.1] libssh2-1.10.0-r4[so:libc.musl-x86_64.so.1]
               make-4.4.1-r1[so:libc.musl-x86_64.so.1] speexdsp-1.2.1-r1[so:libc.musl-x86_64.so.1]
               libedit-20221030.3.1-r1[so:libc.musl-x86_64.so.1]
               libxdmcp-1.1.4-r2[so:libc.musl-x86_64.so.1] zstd-libs-1.5.5-r4[so:libc.musl-x86_64.so.1]
               libgcc-12.2.1_git20220924-r10[so:libc.musl-x86_64.so.1]
               libssl3-3.1.1-r3[so:libc.musl-x86_64.so.1] libexpat-2.5.0-r1[so:libc.musl-x86_64.so.1]
               orc-0.4.34-r0[so:libc.musl-x86_64.so.1]
               mosquitto-libs-2.0.15-r3[so:libc.musl-x86_64.so.1]
               libxxhash-0.8.2-r0[so:libc.musl-x86_64.so.1]
               perl-io-tty-1.17-r1[so:libc.musl-x86_64.so.1]
               networkmanager-cli-1.42.8-r0[so:libc.musl-x86_64.so.1]
               libatomic-12.2.1_git20220924-r10[so:libc.musl-x86_64.so.1]
               openssh-client-default-9.3_p2-r0[so:libc.musl-x86_64.so.1]
               libidn2-2.3.4-r1[so:libc.musl-x86_64.so.1] xz-libs-5.4.3-r0[so:libc.musl-x86_64.so.1]
               dbus-libs-1.14.8-r0[so:libc.musl-x86_64.so.1] popt-1.19-r2[so:libc.musl-x86_64.so.1]
               openssh-keygen-9.3_p2-r0[so:libc.musl-x86_64.so.1]
               pcre2-10.42-r1[so:libc.musl-x86_64.so.1] libxml2-2.11.4-r0[so:libc.musl-x86_64.so.1]
               libverto-0.3.2-r2[so:libc.musl-x86_64.so.1] c-ares-1.19.1-r0[so:libc.musl-x86_64.so.1]
               scanelf-1.3.7-r1[so:libc.musl-x86_64.so.1] gmp-6.2.1-r3[so:libc.musl-x86_64.so.1]
               opus-1.4-r0[so:libc.musl-x86_64.so.1]
               mosquitto-clients-2.0.15-r3[so:libc.musl-x86_64.so.1] jq-1.6-r3[so:libc.musl-x86_64.so.1]
               libarchive-3.6.2-r2[so:libc.musl-x86_64.so.1] libasyncns-0.8-r1[so:libc.musl-x86_64.so.1]
               libncursesw-6.4_p20230506-r0[so:libc.musl-x86_64.so.1]
               dialog-1.3.20230209-r1[so:libc.musl-x86_64.so.1]
               libformw-6.4_p20230506-r0[so:libc.musl-x86_64.so.1]
               libunistring-1.1-r1[so:libc.musl-x86_64.so.1] openssl-3.1.1-r3[so:libc.musl-x86_64.so.1]
               alsa-plugins-pulse-1.2.7.1-r2[so:libc.musl-x86_64.so.1]
               openssh-server-9.3_p2-r0[so:libc.musl-x86_64.so.1]
               openssh-9.3_p2-r0[so:libc.musl-x86_64.so.1]
  libcurl-8.2.0-r1:
    breaks: curl-dev-8.2.1-r0[libcurl=8.2.1-r0]
    satisfies: curl-8.2.0-r1[libcurl=8.2.0-r1] curl-8.2.0-r1[so:libcurl.so.4]
               git-2.40.1-r0[so:libcurl.so.4] cmake-3.26.5-r0[so:libcurl.so.4]
               networkmanager-1.42.8-r0[so:libcurl.so.4]

**Comment by sylvestercavalli** [comment]
Hey! Thanks for your work! 
However I am stuck trying to install it. Do you care to help me?

![Skärmklipp45](https://github.com/gllmlbrt/Alsavo-HassOS/assets/141055662/6e622c2a-6a6d-4cef-bdc4-482352f59fe0)

**Reply by gllmlbrt** [reply]
Hello. I am unfortunately not a CMake expert, but can you confirm you change LOGURU_STACKTRACES in line 97 to 0. So it reads: ‘#define LOGURU_STACKTRACES 0’ before you proceeded with the compilation ?
